.circleci/config.yml

```diff
@@ -84,18 +84,18 @@ jobs:
       - restore_cache:
           name: Restore Yarn cache
           keys:
-            - build-yarn-cache-v4--{{ checksum "yarn.lock" }}
+            - build-yarn-2-cache-v1--{{ checksum "yarn.lock" }}
       - run:
           name: Install dependencies
-          command: yarn install --frozen-lockfile --cache-folder ~/.cache/yarn
+          command: yarn install --immutable
       - run:
           name: Bootstrap
           command: yarn bootstrap --core
       - save_cache:
           name: Save Yarn cache
-          key: build-yarn-cache-v4--{{ checksum "yarn.lock" }}
+          key: build-yarn-2-cache-v1--{{ checksum "yarn.lock" }}
           paths:
-            - ~/.cache/yarn
+            - ~/.yarn/berry/cache
```

Comment:
Yarn wont clear the global cache unless you tell it to so this will grow forever, I suggest setting
```
YARN_ENABLE_GLOBAL_CACHE=false
YARN_ENABLE_MIRROR=false
```
as env variables and caching `.yarn/cache` instead, which we do clean.